lib/ae_mdw/db/sync/name.ex

```diff
@@ -251,7 +252,8 @@ defmodule AeMdw.Db.Sync.Name do
     m_name_exp = Model.expiration(index: {expire, plain_name})
     m_owner = Model.owner(index: {owner, plain_name})
     m_name_owner_deactivation = Model.owner_deactivation(index: {owner, expire, plain_name})
-    %{tx: %{name_fee: name_fee}} = read_raw_tx!(state, txi)
+    name_claim_tx = DbUtil.read_node_tx(state, txi_idx)
```

Comment:
this is a minor, a refactoring opportunity to use pipes since `name_claim_tx` is not used elsewhere

Reply:
Done